lib/ufe/Global.cpp

```diff
@@ -69,6 +69,14 @@ StagesSubject::Ptr g_StagesSubject;
 
 bool InPathChange::fInPathChange = false;
 
+void refreshStages()
+{
+	if(g_StagesSubject)
+	{
+		g_StagesSubject->afterOpen();
```

Comment:
So... When you change the file path on a proxy shape, UFE waits for a notice from usd, which only seems to be sent once maya is idle. This works fine for VP2 selection in the UI, but breaks our unit tests (the stage map is only set up AFTER the tests have finished running).